docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,376 @@
+---
+title: TLS for Kubernetes
+order: 0
+---
+
+# TLS for Kubernetes
+
+Talking over the internet with no encryption is something unthinkable these days. With Redpanda you can help secure your data by enabling [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) across your clusters, with just a few configurations. 
+
+If you haven’t, check our guide about [Encryption, authorization & authentication](https://vectorized.io/docs/acls/).
+
+Here we will dive deeper into how we can configure TLS inside a Kubernetes cluster.
+
+The Custom Resource Definition (CRD) of a Redpanda cluster includes three APIs: Kafka, Admin, and Pandaproxy. Each API can support TLS for server authentication and mutual TLS (mTLS) for client authentication.
+
+TLS allows us to establish a secure connection between a client and a broker in which communication is encrypted. 
+
+mTLS, or 2-way TLS, is an authentication method in which the client and the server must both present certificates. This way both can decide whether the other party can be trusted by ensuring they are who they claim to be.
+
+## Getting started
+
+To demonstrate how to get up and running with TLS in Kubernetes, we’ll create a sample configuration file for a one-node cluster. We'll call it `tls_example.yaml` but feel free to name it however you like. You can save it wherever your Kubernetes network can read it from.
```

Comment:
>You can save it wherever your Kubernetes network can read it from.
The kubernetes config file just needs to be available for `kubectl apply -f` right, not on the network of the kubernetes cluster as far as I know?